warehouse/models/staging/hqta/_hqta.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 
 sources:
   - name: hqta_external_tables
-    database: "{{ target.database }}" # TODO: change me back when deployed
+    database: "{{ var('SOURCE_DATABASE') }}" # TODO: change me back when deployed
```

Comment:
what is the TODO here referring to

Reply:
Oh I'll remove that; I'd used `target.database` as a pre-SOURCE_DATABASE way to test external tables.